A2rchi/chains/prompts.py

```diff
@@ -1,27 +1,31 @@
 # flake8: noqa
 from langchain.prompts.prompt import PromptTemplate
+from A2rchi.utils.config_loader import Config_Loader
 
-condense_history_template = """Given the following conversation between you (the AI named A2rchi), a human user who needs help, and an expert, and a follow up question, rephrase the follow up question to be a standalone question, in its original language.
+config = Config_Loader().config["chains"]["prompts"]
 
-Chat History:
-{chat_history}
-Follow Up Input: {question}
-Standalone question:"""
+def read_prompt(prompt_name, is_condense_prompt=False, is_main_prompt=False):
+    with open(f"config/prompts/{prompt_name}", "r") as f:
```

Comment:
Minor nit: I would make the argument something like `prompt_filepath` (or `prompt_file`) and then pass that into `open()`, (i.e. `with open(prompt_filepath, 'r') as f:`). Then in the config I would provide the relative (or ideally absolute) path to the file. This way if a future developer reads the config they know exactly where that file is supposed to be in the filesystem, without having to read the code.